attestation-service/attestation-service/src/bin/restful/mod.rs

```diff
@@ -44,6 +44,12 @@ pub struct AttestationRequest {
     policy_ids: Vec<String>,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ChallengeRequest {
+    tee: String,
+    tee_params: String,
+}
```

Comment:
BTW, we might need to refactor this request struct. As we might have more abstract contexts apart from `tee` and `tee_params`, like
```rust
#[derive(Debug, Serialize, Deserialize)]
pub struct ChallengeRequest {
    #[flatten]
    inner: HashMap<String, String>,
}
```

Reply:
Fixed, thanks!